trtools/__init__.py

```diff
@@ -1,2 +1,4 @@
-from .version import __version__
-
+try:
+    from .version import __version__
+except ModuleNotFoundError:
+    __version__ = "unknown"
```

Comment:
This is a bit concerning. When does this come up?

Reply:
it shouldn't. If at all, it _might_ happen during development (when you do `pip install -e`) - but even then, setuptools should generate a temporary version for us

Reply:
I can remove this line if you'd like, but I think it will make it easier for us to bypass any issues that might (but shouldn't) arise from setuptools not properly generating a temporary version number during development

Reply:
also, it *will* happen if someone does `pip install .` (without the `-e`) on a local clone of the repo, but nobody should be doing that, as far as I know

Reply:
Okay. Seems good.